tests/manual/monkey/build.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright © 2016-2023 Andy Goryachev <andy@goryachev.com> -->
+<!-- original source: https://github.com/andy-goryachev/FxTextEditor/blob/master/build.xml -->
```

Comment:
Delete these two lines. The original source is not relevant, and any copyright should say "Oracle" (although this file probably doesn't need one).